librad/src/git/storage.rs

```diff
@@ -159,10 +159,6 @@ impl Storage {
 
         // FIXME: properly verify meta
```

Comment:
Can these FIXMEs/TODOs go with this change?

Reply:
We are almost there: I need to implement adding all the needed parents to entity revision commits.